server/cli/cli.go

```diff
@@ -69,12 +69,11 @@ var cockroachCmd = &cobra.Command{
 
 func init() {
 	cockroachCmd.AddCommand(nodeCmds...)
-	cockroachCmd.AddCommand(certCmds...)
```

Comment:
rather than adding `certCmd` to `nodeCmds` add `certCmd` to the arguments to `AddCommand` below